tests/TestCase/View/Helper/PaginatorHelperTest.php

```diff
@@ -842,7 +842,7 @@ public function testGenerateUrlWithPrefixes()
 
         $result = $this->Paginator->sort('name', null, ['url' => $options]);
         $expected = [
-            'a' => ['href' => '/members/posts/index?page=2&amp;sort=name&amp;direction=asc'],
+            'a' => ['href' => '/members/posts/index?sort=name&amp;direction=asc'],
```

Comment:
Do you think these tests are obvious enough? I know that I would be likely to miss this given it is inside another test case method.

Reply:
I'll take a closer look at the test, these changes were the minimum to get 0 fails :)

Reply:
I have added new tests to clarify/highlight expectations.